src/box/memtx_tx.c

```diff
@@ -1932,6 +1958,24 @@ memtx_tx_tuple_clarify_impl(struct txn *txn, struct space *space,
 	return result;
 }
 
+/**
+ * Helper of @sa memtx_tx_tuple_clarify.
+ * Detect and return is_prepared_ok flag.
+ */
+static bool
+detect_whether_prepared_ok(struct txn *txn)
```

Comment:
Please consider using common terminology across our code and transaction specifications, e.g. `is_read_commited_isolation`.

Reply:
Actually in our terminology the isolation level can be `best-effort`. This flag is a bit different and should not be called in that way.